src/components/Blocks/ProjectNav/ProjectNav.tsx

```diff
@@ -0,0 +1,21 @@
+﻿import Image from 'next/image';
+import styles from './ProjectNav.module.scss';
+import { useEffect, useMemo, useState } from 'react';
+
+export default function ProjectNav(props): JSX.Element {
+    return (
+        <div className={props.text == 'Next' ? styles.flippedcontainer : styles.container}>
```

Comment:
I think a way to help with the props and what they mean might be to use object deconstruction: https://www.geeksforgeeks.org/destructuring-of-props-in-reactjs/

Reply:
thanks for the article Ive implemented de-structuring in the new commit